docs/IntegrationGuide.md

```diff
@@ -0,0 +1,117 @@
+# Integration guide
+
+## Terminology
+
+- **checkout-order**: an order which customers create during the checkout. It may contain one or more subscriptions.
+- **template-order**: an order which manages a single subscription and is used as a template to create a
+  subscription-order.
+
+## Before you begin
+
+Create all the required custom types and states located in [resources folder](./resources).
+
+## Step 1: Create checkout order with subscriptions
```

Comment:
```suggestion
## Step 1: Create checkout-orders with subscriptions
```

Reply:
I would use `checkout order`. `checkout` is an adjective describing the noun `order`, same as `template order` or `big order`.  No need to put `-` in between.